src/main/java/racing/domain/Car.java

```diff
@@ -0,0 +1,48 @@
+package racing.domain;
+
+public class Car implements Comparable<Car> {
+    public static final int MAX_NAME_LENGTH = 5;
+    public static final int MINIMUM_FORWARD_NUMBER = 4;
+    private final String name;
+    private int position;
```

Comment:
```suggestion
    public static final int MAX_NAME_LENGTH = 5;
    public static final int MINIMUM_FORWARD_NUMBER = 4;
    
    private final String name;
    private int position;
```
엄격하게 정해진 컨벤션은 아니지만, static 한 상수와 변수를 분리시켜 놓으면 저는 더 눈에 잘 들어오더라구요!